src/modules/lightning_cnn.py

```diff
@@ -164,21 +142,14 @@ def test_step(  # pylint: disable=arguments-differ
         y_reg = y_reg.view(1, -1)  # shape=(1, N_out)
         y_cls = y_cls.view(1, -1)  # shape=(1, N_out)
 
-        if not self.classif:
-            y_reg_hat, y_cls_hat = self.cnn(x)
-            test_mse_loss = self.weighted_mse_loss(y_reg_hat, y_reg)
-            self.accuracy(torch.sigmoid(y_cls_hat), y_cls)
-            metrics = {"test_mse_loss": test_mse_loss, "test_accuracy": self.accuracy}
-            y_train_reg_hat, y_train_cls_hat = self.cnn(x_train)
-        else:
-            y_reg_hat = self.cnn(x)
-            y_cls_hat = torch.zeros(1)
-            test_mse_loss = self.weighted_mse_loss(y_reg_hat, y_reg)
-            metrics = {
-                "test_mse_loss": test_mse_loss,
-            }
-            y_train_reg_hat = self.cnn(x_train)
-            y_train_cls_hat = torch.zeros(1)
+        y_reg_hat = self.cnn(x)
+        y_cls_hat = torch.zeros(1)
```

Comment:
This is 0 for collimator=0 deg, and 1 for collimator=5/355 deg. You can set `y_cls_hat` and `y_train_cls_hat` to either 0 or 1 based on `COLL_5_355`.

Then, these are passed to `plot_img`. Change this [line](https://github.com/nlambriICH/tmi-isocenter/blob/main/src/utils/visualization_cnn.py#L501) by removing the sigmoid function (we don't need it as we already have either 0 or 1).